purescript-cabal-parser/src/Main.nix

```diff
@@ -0,0 +1,2 @@
+
+{ mySubstring = builtins.substring; }
```

Comment:
This is just an example FFI file.

PureScript enforces that an FFI file for a PureScript module is named the same (with a different extension).

`Main.purs` has a `foreign import mySubstring`, so we need to define a `mySubstring` value in this file.

We have to decide how exactly we want the FFI files to interact with the code that `purenix` is generating.  Although since this is just a prototype, we could do something quick and dirty, like hard-code the interaction or the generation of the FFI files.